app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +61,25 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def job_applications
+    @vacancy = VacancyPresenter.new(@vacancy)
+    @application_statuses = vacancy_applications.group_by { |k| k[:status] }.transform_values(&:count)
+  end
+
   private
 
+  def vacancy_applications
```

Comment:
Can this be called `job_application` like we do elsewhere?

Reply:
ill call it applications as job_applications is the name of the route method?

Reply:
Maybe it's better the other way round: the route called 'applications' and the helper 'job_applications'? I don't know. Just in service of nice-looking urls.

Reply:
i agree about nice looking url, ive been round and round in circles with this and it didnt work for some reason which i cant even remeber! ill try again